app/esper/rekall_vgrid.py

```diff
@@ -0,0 +1,354 @@
+"""Utilities for building VGrid inputs from Rekall interval sets
+
+VGrid displays a list of VBlocks. Each VBlock can have multiple tracks, and
+each track shows a timeline of intervals in that track, as well as drawing
+objects and showing metadata of the interval as configured.
+
+There are two main use cases of VGrid:
+    1. VideoVBlocks. This mode displays a VBlock for each video. Each track
+        in the VBlock is an IntervalSet to visualize in that video.
+    2. IntervalVBlocks. This mode displays a VBlock for each interval/clip.
+        Each track in the VBlock is an IntervalSet to visualize in that
+        interval/clip.
+
+The VBlocksBuilders in this module helps building the JSON structure for this
```

Comment:
Thanks for the comprehensive docs btw! I can tell who among is us the former Google engineer ;-)